vive-osc-sender/LighthouseTracking.cpp

```diff
@@ -484,78 +441,23 @@ void LighthouseTracking::ParseTrackingFrame(int filterIndex) {
 	vr::VRActiveActionSet_t actionSet = { 0 };
 	actionSet.ulActionSet = m_actionSetDemo;
 	inputError = vr::VRInput()->UpdateActionState(&actionSet, sizeof(actionSet), 1);
-	if (inputError == vr::VRInputError_None) {
-		sprintf_s(buf, sizeof(buf), "UpdateActionState(): Ok\n");
-		printf_s(buf);
-	}
-	else {
+	if (inputError != vr::VRInputError_None) {
 		sprintf_s(buf, sizeof(buf), "UpdateActionState(): Error: %d\n", inputError);
 		printf_s(buf);
 	}
 
-	/*// Get analog data
-	vr::InputAnalogActionData_t analogData;
-	inputError = vr::VRInput()->GetAnalogActionData(m_actionAnalogInput, &analogData, sizeof(analogData), vr::k_ulInvalidInputValueHandle);
-	if (inputError == vr::VRInputError_None)
-	{
-		sprintf_s(buf, sizeof(buf), "%s | GetAnalogActionData() Ok\n", actionDemoAnalogInputPath);
-		printf_s(buf);
-
-		if (analogData.bActive) {
-			float m_vAnalogValue0 = analogData.x;
-			float m_vAnalogValue1 = analogData.y;
-			sprintf_s(buf, sizeof(buf), "%s | x: %f  y:%f\n", actionDemoAnalogInputPath, m_vAnalogValue0, m_vAnalogValue1);
-			printf_s(buf);
-		}
-		else {
-			sprintf_s(buf, sizeof(buf), "%s | action not avail to be bound\n", actionDemoAnalogInputPath);
-			printf_s(buf);
-		}
-	}
-	else {
-		sprintf_s(buf, sizeof(buf), "%s | GetAnalogActionData() Not Ok. Error: %d\n", actionDemoAnalogInputPath, inputError);
-		printf_s(buf);
-	}
-
-
-	// Get digital data
-	vr::InputDigitalActionData_t digitalData;
-	inputError = vr::VRInput()->GetDigitalActionData(m_actionHideCubes, &digitalData, sizeof(digitalData), vr::k_ulInvalidInputValueHandle);
-	if (inputError == vr::VRInputError_None)
-	{
-		sprintf_s(buf, sizeof(buf), "%s | GetDigitalActionData() Ok\n", actionDemoHideCubesPath);
-		printf_s(buf);
-
-		if (digitalData.bActive) {
-			bool m_vDigitalValue0 = digitalData.bState;
-			sprintf_s(buf, sizeof(buf), "%s | State: %d\n", actionDemoHideCubesPath, m_vDigitalValue0);
-			printf_s(buf);
-		}
-		else {
-			sprintf_s(buf, sizeof(buf), "%s | action not avail to be bound\n", actionDemoHideCubesPath);
-			printf_s(buf);
-		}
-	}
-	else {
-		sprintf_s(buf, sizeof(buf), "%s | GetDigitalActionData() Not Ok. Error: %d\n", actionDemoHideCubesPath, inputError);
-		printf_s(buf);
-	}*/
-
-	
 	// get pose data
 	vr::InputPoseActionData_t poseController;
 	inputError = vr::VRInput()->GetPoseActionDataRelativeToNow(m_actionDemoController, vr::TrackingUniverseStanding, 0, &poseController, sizeof(poseController), vr::k_ulInvalidInputValueHandle);
 
 	if (inputError == vr::VRInputError_None) {
-		sprintf_s(buf, sizeof(buf), "\n%s | GetPoseActionData() Ok\n", actionDemoControllerPath);
-		printf_s(buf);
+		//sprintf_s(buf, sizeof(buf), "\n%s | GetPoseActionData() Ok\n", actionDemoControllerPath);
+		//printf_s(buf);

```

Comment:
Best not to leave commented out code. Is this intentional? If so, do one of:
- Leave another comment explaining why it is commented out
- Just remove it alltogether